Segmentation violation

Just installed `todoist` (built manually at 2a99599) and after entering my API token, I ran the `list` command and get the following error:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x58 pc=0x145595c]

goroutine 1 [running]:
main.List.func1(0x0, 0x0)
        /Users/seanzellmer/go/src/github.com/sachaos/todoist/list.go:40 +0xfc
main.traverseItems(0x0, 0xc000138d98, 0x0)
        /Users/seanzellmer/go/src/github.com/sachaos/todoist/list.go:9 +0x3e
main.List(0xc0001be000, 0x0, 0x0)
        /Users/seanzellmer/go/src/github.com/sachaos/todoist/list.go:35 +0x2d8
github.com/urfave/cli.HandleAction(0x14b1c80, 0x156c378, 0xc0001be000, 0xc0001be000, 0x0)
        /Users/seanzellmer/go/src/github.com/urfave/cli/app.go:523 +0xbe
github.com/urfave/cli.Command.Run(0x154ad81, 0x4, 0x0, 0x0, 0xc00016a010, 0x1, 0x1, 0x155131a, 0xe, 0x0, ...)
        /Users/seanzellmer/go/src/github.com/urfave/cli/command.go:174 +0x521
github.com/urfave/cli.(*App).Run(0xc000164000, 0xc00009e040, 0x2, 0x2, 0x0, 0x0)
        /Users/seanzellmer/go/src/github.com/urfave/cli/app.go:276 +0x71c
main.main()
        /Users/seanzellmer/go/src/github.com/sachaos/todoist/main.go:262 +0x1555
```

**Info**
- `go version go1.12.9 darwin/amd64`
- OS : OSX 10.11.6
- Built manually via go (though these errors happened via homebrew too)

Let me know if you need any other information! I have no experience with Go so do not know where to start debugging this error.

--- a/list.go
+++ b/list.go
@@ -1,8 +1,10 @@
 package main
 
 import (
+	"fmt"
 	"github.com/sachaos/todoist/lib"
 	"github.com/urfave/cli"
+	"os"
 )
 
 func traverseItems(item *todoist.Item, f func(item *todoist.Item, depth int), depth int) {
@@ -31,6 +33,11 @@ func List(c *cli.Context) error {
 
 	itemList := [][]string{}
 	rootItem := client.Store.RootItem
+
+	if rootItem == nil {
+		fmt.Fprintln(os.Stderr, "There is no task. You can fetch latest tasks by `todoist sync`.")
+		return nil
+	}
 
 	traverseItems(rootItem, func(item *todoist.Item, depth int) {
 		r, err := Eval(ex, item, client.Store.Projects, client.Store.Labels)

